web/src/compose_validate.ts

```diff
@@ -725,10 +725,13 @@ export function check_overflow_text(): number {
     const max_length = realm.max_message_length;
     const remaining_characters = max_length - text.length;
     const $indicator = $("#compose-limit-indicator");
+    const $textarea = $("textarea#compose-textarea");
 
     if (text.length > max_length) {
+        $indicator.removeClass("approaching_limit");
```

Comment:
We're generally trying to move classes to use hyphens rather than underscores. This could be an opportunity to fix up `over_limit`, too. See https://zulip.readthedocs.io/en/latest/subsystems/html-css.html#use-clear-unique-names-for-classes-and-object-ids